cli/src/printer.rs

```diff
@@ -98,11 +98,19 @@ impl DisplayTable for Bucket {
 
 impl DisplayTable for Quota {
     fn to_table_headers() -> Row {
-        row![bFg => "Kind", "Hard Limit"]
+        row![bFg => "Kind", "Hard Limit", "Usage (Total)", "Usage %"]
     }
 
     fn to_table_row(&self) -> Row {
-        row![self.quota_kind, Thousands(self.hard_limit)]
+        row![
+            self.quota_kind,
+            Thousands(self.hard_limit),
+            Thousands(self.current_max_usage),
+            format!(
+                "{:.0}%",
+                (self.current_max_usage as f64 / self.hard_limit as f64) * 100.0
+            )
```

Comment:
You probably want edge case handling for if `self.hard_limit == 0`, otherwise you'll get `NaN%` here, in the divide by zero case

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d2ddc482f0429345cbef5436efee6bd9